_posts/2025-03-31-mmaler-blogpost-1-intro.adoc

```diff
@@ -0,0 +1,133 @@
+Companies looking to reduce cloud costs while maintaining high performance will find Quarkus an excellent solution.
+
+=== Reactive at its core
+
+At its core, Quarkus is built on Eclipse Vert.x, a high-performance reactive toolkit.
+Still, it allows developers to work primarily in an imperative style while leveraging the performance benefits of its reactive underpinnings.
+This hybrid approach allows developers to squeeze out even more efficiency from traditional imperative programming while offering the flexibility to adopt reactive patterns where they make sense.
+Unlike traditional reactive-only frameworks, Quarkus enables developers to combine both imperative and reactive styles in a single application.
+This is particularly beneficial for systems requiring high throughput and low latency, ensuring that applications remain robust under heavy load.
+Quarkus's reactive model makes it ideal for event-driven architectures and microservices.
+
+
+== Which of your current development pains could Quarkus solve effortlessly?
+
+In general, Quarkus will save you a lot of time and money as well as CPU and memory resources.
+Besides this, lets mention Dev Services and geared extension ecosystem.
+
+=== Dev Services
+
+Quarkus Dev Services reduce friction during development and testing by automatically provisioning required services such as databases, message brokers, or identity providers.
+For example, if your application includes PostgreSQL, Kafka, or Keycloak extensions, Quarkus can spin up the necessary containers without any manual setup.
+This allows you to focus on coding instead of configuring infrastructure, accelerating your local development workflow.
+
+=== Vast extension ecosystem
+
+Quarkus offers a rich extension ecosystem that simplifies integration with essential technologies such as databases, messaging systems, authentication providers, and cloud services.
+
+Popular extensions include:
+* `quarkus-hibernate-orm` and `quarkus-jdbc-postgresql` for seamless data persistence.
+* `quarkus-smallrye-reactive-messaging` and `quarkus-kafka-client` for reactive messaging and Apache Kafka integration.
+* `quarkus-oidc` for implementing OpenID Connect authentication and securing applications.
+* `quarkus-micrometer` and `quarkus-opentelemetry` for observability, metrics, and tracing.
+* `quarkus-container-image-docker` and `quarkus-kubernetes` for containerization and deployment to Kubernetes platforms.
+
+These extensions are widely adopted because they reduce boilerplate, provide reliable default configurations out of the box, and follow cloud-native best practices—making it easy to plug Quarkus into real-world architectures.
+
+== Even better Quarkus? Here’s what new tools are on the way
+
+=== Panache 2: Simplifying data access
+
+Hibernate ORM with Panache is a lightweight abstraction layer built on top of Hibernate ORM that simplifies data interactions and reduces boilerplate code.
```

Comment:
cool to mention this but without code or some examples this to me does not provide much value to a reader ? its a lot of positive words for sure; but what have the reader learned after reading this?

Reply:
Can add a link to a video presentation, maybe.

Reply:
@cescoffier Are you for keeping it?

Reply:
I took it down.